src/libs/actions/ReimbursementAccount/index.js

```diff
@@ -1,6 +1,6 @@
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../../ONYXKEYS';
-import setupWithdrawalAccount from './setupWithdrawalAccount';
+import {setupWithdrawalAccount, connectBankAccountManually} from './setupWithdrawalAccount';
```

Comment:
Let's have `connectBankAccountManually` in [BankAccount.js](https://github.com/Expensify/App/blob/main/src/libs/actions/BankAccounts.js). We're thinking of eventually deprecating setupWithdrawalAccount.js